Modelica/Electrical/Analog/Semiconductors/Thyristor.mo

```diff
@@ -24,6 +24,7 @@ model Thyristor "Simple Thyristor Model"
   SI.Voltage vControl(start=0);
   SI.Voltage vContot;
   SI.Voltage vConmain;
+  SI.Voltage vRef = 0.65;
```

Comment:
```suggestion
  parameter SI.Voltage vRef = 0.65;
```
Or possibly constant, or final parameter.
Having a normal variable with binding is kind of unusual.